src/backend/tasks_io/datafeeds/parsers/fms_api/fms_api_match_parser.py

```diff
@@ -121,9 +120,19 @@ def parse(
                     key=lambda team: team["station"],
                 )
             )
+
+            null_team = any(t["teamNumber"] is None for t in sorted_teams)
+            if (
+                null_team
+                and match["scoreRedFinal"] is None
+                and match["scoreBlueFinal"] is None
+            ):
+                continue
+
```

Comment:
This logic moves up, so we can more clearly skip over unplayed matches without teams configured